### ISSUE TITLE
Android Bootcamp Project 0 Read For Review

### ISSUE THREAD

**Issue by didiimnida** [didiimnida/todo#1]
My app is complete, please review @codepath, @codepathreview

**Comment by nidhi1608** [comment]
Diana,

Thanks for applying and submitting your pre-work. Based on your LinkedIn it is unclear if you meet the [selection criteria for the course](http://courses.codepath.com/snippets/intro_to_android/selection_criteria). In particular:
- 2+ years of professional software development experience (average participant has 5+ years)
- Bachelor's degree in CS/Engineering (if not then 4+ years of professional software experience)
- Sustained professional software experience at a company for at least 2 years is strongly preferred

Can you review that and let me know if you can confirm that you meet the requirements? If not, I can add you as a "remote self-paced observer" so you can participate in the program and learn at your pace. In the meantime, you should also check out the [resources we provide here](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me).

**Comment by didiimnida** [comment]
Yes, most of my professional experience is within academia.  I have worked on coding projects within research projects at University of Michigan - Ann Arbor and Nanyang Technological University in various engineering departments in these institutions.  Hence, I have only been in the "technology industry" proper less than 2 years due to being stationed in academic situations.

**Comment by nidhi1608** [comment]
Thanks for the detailed notes Diana. Happy for you have to applied and that you submitted the assignment here. Note that the course is focused on people of a particular number of years of professional experience because this allows for the course to move significantly faster and for groups of people to work together significantly smoother. Any program that combines people of significantly different professional experience levels tends to be considerably less effective. We've seen this ourselves many times in past iterations.

I am happy to have you as a "remote self-paced observer" so you can participate in the program and learn at your pace. In the meantime, you should also check out the [resources we provide here](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me). Let me know if you'd like to be a remote observer.

**Comment by didiimnida** [comment]
Will the class sessions be uploaded to your site for the "remote self-paced observer" portion?

**Comment by nidhi1608** [comment]
Yes, we already have recorded sessions from the past on the courses portal that you'll get access to should you choose to be a remote observer and when the content for a session changes, we record that session and make it available to all participants as well. Hope that answers your question.
